pyatv/raop/__init__.py

```diff
@@ -285,11 +285,13 @@ async def stream_file(self, file: Union[str, io.BufferedReader], **kwargs) -> No
 
         INCUBATING METHOD - MIGHT CHANGE IN THE FUTURE!
         """
+        audio_file = None
```

Comment:
Yes old stuff. Without this change a variable is not defined error instead of an `AuthenticationError` was thrown. I saw that you already added this change.